transports/janus_mqtt.c

```diff
@@ -306,6 +350,25 @@ int janus_mqtt_init(janus_transport_callbacks *callback, const char *config_path
 	}
 
 	/* Connect configuration */
+	janus_config_item *mqtt_version = janus_config_get(config, config_general, janus_config_type_item, "mqtt_version");
+	const char *mqtt_version_str = (mqtt_version && mqtt_version->value) ? mqtt_version->value : JANUS_MQTT_VERSION_DEFAULT;
+
+	if(strcmp(mqtt_version_str, JANUS_MQTT_VERSION_3_1) == 0) {
+		ctx->connect.mqtt_version = MQTTVERSION_3_1;
+	} else if(strcmp(mqtt_version_str, JANUS_MQTT_VERSION_3_1_1) == 0) {
+		ctx->connect.mqtt_version = MQTTVERSION_3_1_1;
+	} else if(strcmp(mqtt_version_str, JANUS_MQTT_VERSION_5) == 0) {
+	#ifdef MQTTVERSION_5
```

Comment:
This block of `#ifdef #else #endif` still has a broken indentation.

Reply:
Why? `#ifdef #else #endif` are aligned one tab left and the code itself is aligned like it would be without these directives.